pkg/cli/mirror/release.go

```diff
@@ -342,3 +365,81 @@ func (d downloads) Merge(in downloads) {
 		d[k] = v
 	}
 }
+
+//go:embed release-configmap.yaml
+var b []byte
+
+func (o *ReleaseOptions) generateReleaseSignatures(releaseDownloads downloads) error {
+
+	httpClientConstructor := sigstore.NewCachedHTTPClientConstructor(o.HTTPClient, nil)
+
+	manifests, err := manifest.ParseManifests(bytes.NewReader(b))
+
+	if err != nil {
+		return err
+	}
+
+	// Attempt to load a verifier as defined by the release being mirrored
+	imageVerifier, err := verify.NewFromManifests(manifests, httpClientConstructor.HTTPClient)
+
+	if err != nil {
+		return err
+	}
+
+	reg := regexp.MustCompile(".*sha256:")
+
+	for image, _ := range releaseDownloads {
+		digest := reg.ReplaceAllString(image,"sha256:")
+
+		ctx, cancelFn := context.WithCancel(context.Background())
+		defer cancelFn()
+		if err := imageVerifier.Verify(ctx, digest); err != nil {
+			logrus.Errorf("An image was retrieved that failed verification: %v", err)
+		}
+
+		cmData, err := verify.GetSignaturesAsConfigmap(digest, imageVerifier.Signatures()[digest])
```

Comment:
When I tested with an OKD release the configMap was created with no `binaryData`. Might be worth doing a check to see if the signature was retrieved before creating the file.

Reply:
Currently I have made this exit after the Verification fails via a break. This means we don't mirror signatures for OKD.